takwimu/templates/takwimu/search_page.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'takwimu/_layouts/page.html' %}
+{% load jsonify staticfiles sass_tags %}
+
+{% block content %}
+  <div id="takwimuSearchResults"></div>
+{% endblock %}
+
+{% block body_javascript_extra %}
+  {{ block.super }}
+
+  <script>
+    var takwimu = window.takwimu || {};
+    takwimu.page = takwimu.page || {};
+    takwimu.page.id = 'search';
+    var search = {};
+    search['searchQuery'] = '{{ search_query }}';
```

Comment:
Since we're using wagtail `Page` models now, can't these variables (search query, etc.) be return as part of `SearchPage` API?

Reply:
Correct: These are initial variables passed from a different page to `/search/` page hence needed.